### DISCUSSION TITLE
Feature Request: Support for Kivy KvLang

### DISCUSSION THREAD

**Discussion by davidtressler** [zed-industries/zed#9097] [Feature Requests]
Hi All 

I'm in love with Zed, can't wait for Linux Support! 

I wonder if it's possible to support kvlang  https://marketplace.visualstudio.com/items?itemName=Monk3yDev.kvlang 

I know it has a pretty small userbase, but I could stop using VsCode if its included. 

Many Thanks

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
